htdocs/fourn/commande/card.php

```diff
@@ -1003,18 +1011,17 @@
 									$lines[$i]->fetch_optionals($lines[$i]->rowid);
 									$array_option = $lines[$i]->array_options;
 								}
-								
-								$res = $productsupplier->find_min_price_product_fournisseur($lines[$i]->fk_product, $lines[$i]->qty);
-								/*if ($productsupplier->id > 0)
-								{
-								    $res = $productsupplier->fetch($productsupplier->id);
-								}*/
-								
+
+								$idprod = $productsupplier->find_min_price_product_fournisseur($lines[$i]->fk_product, $lines[$i]->qty);
+								$res = $productsupplier->fetch($idprod);
```

Comment:
Here idprod is always 1 or -1 because it is return of find_min_price_product_fournisseur()
Also $idprod var is useless. It is already $lines[$i]->fk_product